.github/workflows/ci.yml

```diff
@@ -379,31 +374,25 @@ jobs:
 
   # Builds Docker image from the core distribution files and uploads to Docker Hub.
   docker:
-    name: Docker (CUDA ${{ matrix.cuda }})
+    name: Docker
     timeout-minutes: 18
     if: github.repository == 'allenai/allennlp'
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        cuda: ['11.3']
 
     steps:
     - uses: actions/checkout@v2
 
     - name: Set image name and torch version
-      env:
-        CUDA: ${{ matrix.cuda }}
       run: |
-        echo "DOCKER_TORCH_VERSION=${TORCH_VERSION}-cuda${CUDA}" >> $GITHUB_ENV;
         if [[ $GITHUB_EVENT_NAME == 'release' ]]; then
-            echo "DOCKER_IMAGE_NAME=allennlp/allennlp:${GITHUB_REF#refs/tags/}-cuda${CUDA}" >> $GITHUB_ENV;
+            echo "DOCKER_IMAGE_NAME=allennlp/allennlp:${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV;
         else
-            echo "DOCKER_IMAGE_NAME=allennlp/commit:${GITHUB_SHA}-cuda${CUDA}" >> $GITHUB_ENV;
+            echo "DOCKER_IMAGE_NAME=allennlp/commit:${GITHUB_SHA}" >> $GITHUB_ENV;
```

Comment:
Per our slack conversation I think it's important that we tag the image with the CUDA version.